lib/project.ts

```diff
@@ -164,7 +164,10 @@ export class Project implements Project.IProject {
 				return !isExcluded && !isSubprojectDir;
 			});
 
-			var ignoreFilesRules = this.$pathFilteringService.getRulesFromFile(path.join(projectDir, Project.IGNORE_FILE));
+			var ignoreFilesRules = <string[]>_(this.ignoreFilesConfigurations)
```

Comment:
It is required, because without it the source does not compile. I think that the issue is due to insufficiently sophisticated typescript definitions.